dluhc-component-library/src/components/siteSelectionForm/components/MapPage.tsx

```diff
@@ -1,3 +1,6 @@
+import { Polygon } from "ol/geom";
+import { useMemo } from "react";
```

Comment:
`Non-blocker` this is coming from React rather than Preact. I don't think we care at this point though

Reply:
Good spot, ill fix it